projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.module.spec.ts

```diff
@@ -78,7 +78,6 @@ describe('NgxSkeletonLoaderModule method', () => {
   });
 
   it('should NOT call console.info() method', () => {
-    // eslint-disable-next-line no-console
```

Comment:
The project should keep avoiding console usage, unless in places enforcing it via `// eslint-disable-next-line`. Could you update the ESLint config to reflect that, please?

Reply:
done 👍

restored `no-console` rule.